Explorer/Assets/Plugins/CommandTerminal/ConsoleBuffer.cs

```diff
@@ -0,0 +1,46 @@
+using System.Collections.Generic;
+using UnityEngine;
+
+// ORIGINAL OPEN SOURCE PLUGIN SCRIPT:
+// https://github.com/stillwwater/command_terminal/blob/0f5918ea79014955b24c7d431625a97a1cac8797/CommandTerminal/CommandLog.cs
+
+namespace CommandTerminal
+{
+    public struct LogItem
```

Comment:
I suggest to make this struct readonly

Reply:
Implemented at https://github.com/decentraland/unity-explorer/pull/1546/commits/0d20de12ee24b27c5ff92c7bcffe190a680f4cd7